code/index.html

```diff
@@ -3,28 +3,98 @@
+        <a href="#" class="nav-list">Community</a>
+      </nav> 
+        
+      <h1>HOW MANY MOUNTAINS DO YOU HAVE IN THE BANK?</h1>
+    </header>
 
+    <main>
     <!-- Signup form -->
+      <form id="form" action="https://httpbin.org/anything" method="POST">
+        
+        <div>
+          <p1>Track your climbs with our community of climbers</p1>
+        </div>
+
+        <label for="email">
+          <input type="email" id="email" name="email" placeholder="Type your email" required />
+        </label>
+
+        <label for="password">
+          <input type="password" id="password" name="password" placeholder="Enter password" required />
+        </label>
+
+        <label for="terms">I agree with the <a href="#" class="terms">terms and conditions</a>
+          <input type="checkbox" id="checkbox1" class="checkbox" name="terms" required />
+        </label>
+
+        <label for="newsletter">Sign up to our newsletter
+        <input type="checkbox" id="checkbox2" class="checkbox" name="newsletter" />
+        </label>
+
+        <button type="submit" id="logIn" class="button">Log in</button>
+        
+        <button type="submit" id="signUp" class="button">Sign up</button>
+      </form>
 
-    <section class="accordion">
       <!-- Accordion -->
-      <h1>FAQ</h1>
+      <section class="accordion">
+        <h2>FAQ</h2>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+        <div class="question" id="section1">Can I track my climbs in an indoor facility?</div>
+        <div class="answer" >
+          <p>Yes! You can use the beginner part of the app to track your indoor climbs.
+            When you get outside make sure to switch to advanced mode.
+          </p>
+        </div>
+
+        <div class="question" id="section2">Can I manually log my climbs?</div>
+        <div class="answer">
+          <p>Of course, we know you can´t bring a phone or fitness tracker everywhere. </p>
+        </div>
+
+        <div class="question" id="section3">How do I get started?</div>
+        <div class="answer">
+          <p>Start with signing up!</p>
+        </div>
+     </section>
+
+    </main>
+    
+    <footer>
+      <div class="made-by">Made by Ida Näslund in Technigo Bootcamp Spring 2022
+      </div> 
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div>
+        <a href="https://github.com/idanaslund" class="social-logo">
+          <i class="fa fa-github-square fa-2x" aria-hidden="true">
+          </i>
+        </a>
+        <a href="https://www.linkedin.com/in/ida-n%C3%A4slund-8bb888170/" class="social-logo">
+          <i class="fa fa-linkedin-square fa-2x" aria-hidden="true">
+          </i>
+        </a>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+    </footer>
+    
+    <script src="script.js"></script>
```

Comment:
Such a pleasure to read this code Ida, very clear, I am struggling to find any constructive feedback!